# joint belief
time stamp
sloop_ros/SloopMosObjectBelief[] object_beliefs